include/benchmark/benchmark.h

```diff
@@ -280,6 +280,10 @@ BENCHMARK(BM_test)->Unit(benchmark::kMillisecond);
 namespace benchmark {
 class BenchmarkReporter;
 
+// Default number of minimum benchmark running time in seconds.
+const double kDefaultMinTime = 0.5;
+const char kDefaultMinTimeStr[] = "0.5s";
```

Comment:
i was hoping to not have this declared twice.  is there no way to set the default based on `kDefaultMinTime` directly?

or extract the double default from the string version?

Reply:
kind of yes :) I've moved the double field to benchmark_runner.cc (where it's used) and parse the string field there.